src/app/components/score-chart/score-chart.component.html

```diff
@@ -1,25 +1,24 @@
-<div class="score-chart-container">
-  <div class="score-chart-number-column">
-    <span class="mat-title">
-      Your {{ label }} Score Today
-    </span>
-
-    <span class="mat-display-3">
-      {{ currentValue ? currentValue : '--' }}
-    </span>
-
-    <span>
-      Ave. {{ averageValue ? averageValue : '--' }}
-    </span>
-  </div>
-
-  <div class="score-chart-highcharts-column">
-    <highcharts-chart
-    [Highcharts]="Highcharts"
-    [options]="chartOptions"
-  >
-  </highcharts-chart>
-  </div>
-</div>
+<mat-card>
+  <mat-card-title>
+    {{ label }} Score
+  </mat-card-title>
+  <mat-card-content class="score-card-content">
+    <div class="score-chart-number-column">
+      <span class="mat-display-3">
+        {{ currentValue ? currentValue : '--' }}
```

Comment:
Edge case question: if `currentValue === 0`, do we want to show `0` or `--`? 
I prefer `0`, because `--` means no data.

To show `0`, you will need to explicitly test for null equality, i.e. `currentValue !== null`; same for averageValue

Reply:
Hmmm make sense. Although having 0 is quite hard in both accent and clarity - the user will need to pronounce every single word wrong. Will change it to a null check then.